tox.ini

```diff
@@ -1,11 +1,14 @@
 [tox]
 envlist = py26,py27,py34,py35,py36,pypy,docs,coverage
 
+[flake8]
+max-line-length=100
+
 [testenv]
 deps =
     pyyaml
-    pytest
-    mock <= 1.0.1
+    pytest==2.7.3
```

Comment:
I had some really strange errors from pytest and pinning it to a previous version solved it. If the CI is fine without it, I don't mind removing the pinning (although, if mock is pinned, pinning pytest to a version that works properly with it isn't necessarily a bad thing)

Reply:
I'd like to avoid pinning it, if we can. There shouldn't really be any interdependence between mock and pytest. 

The reason mock is pinned to a max version is because they dropped python 2.6 support after that version, and CI still tests against 2.6. I think it might be fine to drop that support now, and unpin `mock`, but we should do that in a separate PR.